linux_os/guide/system/software/integrity/software-integrity/aide/aide_scan_notification/oval/shared.xml

```diff
@@ -5,9 +5,20 @@
     <criteria operator="AND">
       <extend_definition comment="Aide is installed" definition_ref="package_aide_installed" />
       <criteria operator="OR">
-        <criterion comment="notify personnel when aide completes" test_ref="test_aide_scan_notification" />
-        <criterion comment="notify personnel when aide completes" test_ref="test_aide_var_cron_notification" />
-        <criterion comment="notify personnel when aide completes in cron.(d|daily|weekly|monthly)" test_ref="test_aide_crontabs_notification" />
+        <criterion comment="notify personnel when aide completes"
+          test_ref="test_aide_scan_notification" />
+        <criterion comment="notify personnel when aide completes"
+          test_ref="test_aide_var_cron_notification" />
+        <criterion comment="notify personnel when aide completes in cron.(d|daily|weekly|monthly)"
+          test_ref="test_aide_crontabs_notification" />
+        <criteria operator="AND">
```

Comment:
Should this new criteria be guarded as SLE-only? The new code in Bash and Ansible remediations is marked as SLE-only.

Reply:
Well I prefer to be SLE only for now and other maintainers to adopt it if consider viable, therefore the bash and ansible remediations aer platform dependant

Reply:
Thanks for reply. But, if you think that this part should be SLE-only for now, would it be better to put it inside a Jinja `if product in ...` block?